configs/solo/solo_r50_fpn_1x_coco.py

```diff
@@ -0,0 +1,14 @@
+_base_ = [
+    '../_base_/datasets/coco_instance.py', '../_base_/models/solo_r50_fpn.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+# optimizer
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
```

Comment:
momentum and weight_decay is unnecessary here because it is the same as the base config.